highlight/highlight
# Add environment to traces

## Summary
Add environment filter for traces

## How did you test this change?
1) Load the traces page
2) View a trace
- [ ] Trace should be created with the appropriate environment
- [ ] Can filter traces with environment

https://www.loom.com/share/827d8ebfd7744e18890f0dd82edf73b6?sid=afa49db8-f53a-426f-ba9b-48b020af45b6

## Are there any deployment considerations?
Make sure backwards compatible for sdks that do not pass in environment

## Does this work require review from our design team?
N/A
